### ISSUE TITLE
Overview map draggable area is sticky if draw interaction is enabled

### ISSUE THREAD

**Issue by igoroctaviano** [openlayers/openlayers#12187]
When the overview map is opened and the draw interaction is activated, the drag of the red area inside the overview map becomes slow and sticky.

Fiddle with bug: http://jsfiddle.net/igoroctaviano/4y0dzn5t/

**Expected behavior**
Drag should be smooth like without drag interaction active.

**Comment by mike-000** [comment]
Your fiddle is using version 5.2 but 6.5 http://jsfiddle.net/f1ogejk8/ has the same problem.

**Comment by igoroctaviano** [comment]
Good catch. I created this one based on an old fork but I have spotted the issue in the actual code which has the newer version. So yes, same problem

**Comment by RobQuincey** [comment]
Just because I was having trouble seeing the issue, I've added a checkbox to your fiddle to turn the line control on and off, so you can quickly see the difference between normal behaviour and 'sticky' behaviour - http://jsfiddle.net/u2gw7hd8/ - Just in case its useful to anyone

**Comment by mike-000** [comment]
A workaround, based on this example https://openlayers.org/en/latest/examples/vector-tile-info.html, is to place the map in an external div, and then place that div on top of the map using css http://jsfiddle.net/4z5vLhtg/
